ecs.tf

```diff
@@ -12,6 +12,7 @@ data "template_file" "ecs_user_data" {
 
 data "aws_ami" "amazon_ecs_ami" {
   most_recent = true
+  owners = ["amazon"]
```

Comment:
Is the `owners-alias` filter needed now?

Reply:
Tested this, it seems to deploy fine without the filter so will remove.